internal/provisioner/model/names.go

```diff
@@ -82,17 +82,19 @@ func (c *Contour) EnvoyRBACNames() RBACNames {
 // ComponentLabels returns labels for a Contour component.
 func (c *Contour) ComponentLabels() map[string]string {
 	labels := map[string]string{
-		"app.kubernetes.io/name":       "contour",
-		"app.kubernetes.io/instance":   c.Name,
-		"app.kubernetes.io/component":  "ingress-controller",
-		"app.kubernetes.io/managed-by": "contour-gateway-provisioner",
+		"app.kubernetes.io/instance": c.Name,
```

Comment:
any particular reason to remove the `name` or `managed-by` labels as things that come by default? I can understand changing the value of the `component` label to something else

Reply:
this is a wrong change, i reverted it